### ISSUE TITLE
Vimium hijacks Ctrl+[, even when [ is in excluded keys

### ISSUE THREAD

**Issue by kenny47** [philc/vimium#3168]
Some sites, e.g. [keep.google.com](https://keep.google.com) use ctrl+[ to unindent. With vimium enabled, and [ in the excluded keys list, I still cannot unindent. Only when I completely disable vimium for the page (which I don't want to do), the keystroke works.

**Comment by timvisee** [comment]
What about blacklisting `<c-[>`? I'm not sure whether that works, but that's how to notate such a binding in Vim.

**Comment by kenny47** [comment]
Tried that, it doesn't work and the top line of the extension's window says "Vimium will exclude -<>[c", so I guess this doesn't do what is expected.

**Comment by gdh1995** [comment]
The command `passNextKey` may help:
https://github.com/philc/vimium/wiki/Tips-and-Tricks#using-the-escape-key-in-inputs

**Comment by mimkorn** [comment]
I have the same issue. @kenny47 I cannot replicate your workaround. What specifically have you done to make it work?

**Comment by kenny47** [comment]
I have this in my mappings: 
mapkey <c-[> <anyOldJunk>

**Comment by mimkorn** [comment]
Just that? Nothing after it? I added it, but it did not help, pressing <c-[> still just defocuses the gmail composer.

**Comment by kenny47** [comment]
Yes, just that. I checked now, it works in keep, tasks, gmail composer.. I don't see anything else relevant in my config.

**Comment by gregorias** [comment]
`mapkey <c-[>` didn't work for me as well but `mapkey <c-[> <junk>` did.
